basic_datatypes.ipynb

```diff
@@ -2491,6 +2841,7 @@
    "cell_type": "markdown",
    "metadata": {},
    "source": [
+    "Important to note: when unpacking you will recieve a list instead of whatever iterable you had to begin with. This may be relevant when further processing data.\n",
```

Comment:
Please keep one sentence per line. Please check the whole "Unpacking" section for this. Many sentences here are written in one line (which makes it harder to review).